blank_project/src/index.html

```diff
@@ -7,11 +7,11 @@
 </head>
 <body style="background: #fff">
   <div class="container">
-    Contract says:
+    Contract says 2:
```

Comment:
@janedegtiareva hmm, seems like typo

Reply:
good catch. I use these sometimes to make sure i am looking at the latest stuff and forget to undo lol